WifiART/src/structs/MapState.java

```diff
@@ -0,0 +1,10 @@
+package structs;
+
+public class MapState {
```

Comment:
This class provides a setter of mapUrl and no other methods. How would a caller use it?

Reply:
Since I haven't yet implemented the encoding, a getter has not been necessary, but according to my current plan, getter methods will be needed for accessing the values in the state. As I will be pursuing a depth-first approach from now on, I think it will be good to add TODOs where classes are incomplete, so I don't forget when I get to those subelements.

Reply:
Yes it would be good to add TODOs so it's clear what the clear is supposed to be used. 

Alternatively you can start from the building blocks (MapState, BssidState) and send them for review first, then glue everything together.